manager/manager.go

```diff
@@ -580,6 +582,7 @@ func (m *Manager) Stop(ctx context.Context, clearData bool) {
 
 	m.raftNode.Cancel()
 
+	m.collector.Stop()
```

Comment:
It looks like `m.collector` is only initialized when the node becomes the leader, so this line is crashing in CI.